src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -291,4 +292,34 @@ private List<Error> validatePackagePoLine(CompositePoLine compPOL) {
 
     return convertErrorCodesToErrors(compPOL, errors);
   }
+
+  public List<Error> validateForBinadryActive(CompositePoLine poLine) {
+    List<Error> errors = new ArrayList<>();
+    if (poLine.getDetails() != null && poLine.getDetails().getIsBindaryActive()) {
+      validateOrderFormatForBindaryActive(poLine, errors);
+      validateCreateInventoryForBindary(poLine, errors);
+    }
+    return errors;
+  }
+
+  private void validateOrderFormatForBindaryActive(CompositePoLine poLine, List<Error> errors) {
+    if (!Objects.equals(poLine.getOrderFormat().value(), PoLine.OrderFormat.PHYSICAL_RESOURCE.value())
+      && !Objects.equals(poLine.getOrderFormat().value(), PoLine.OrderFormat.P_E_MIX.value())) {
+      var param = new Parameter().withKey("orderFormat").withValue(poLine.getOrderFormat().value());
+      var error = new Error().withMessage("When PoLine is bindary active, its format type must be '" +
```

Comment:
better to use error code also as we do in other places

Reply:
Added error codes